### PR TITLE
Remove Tizen 4.0/5.0 bindings

### PR DESCRIPTION
**Pull request by JSUYA**
The minimum supported version of flutter-tizen is 5.5.
Therefore, remove the 4.0 and 5.0 bindings.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.2.7
```
Comment (swift-kim) [inline CHANGELOG.md:1]: We're removing public APIs.
```suggestion
## 0.3.0
```

Comment (JSUYA) [reply CHANGELOG.md:5]: Thanks. I updated version and updated callbacks package

File: README.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ To use this package, add `ffi` and `tizen_interop` as dependencies in your `pubs
 ```yaml
 dependencies:
   ffi: ^2.0.1
-  tizen_interop: ^0.2.7
+  tizen_interop: ^0.3.0
```
Comment (swift-kim) [inline README.md:15]: Please update the `pubspec.yaml` file.

Comment (JSUYA) [reply README.md:15]: thank you for your check. i updated it

Comment (swift-kim) [reply README.md:15]: Also make sure to publish the package on pub.dev after merge. It should be done manually by triggering the [release](https://github.com/flutter-tizen/tizen_interop/actions/workflows/release.yml) workflow (`workflow_dispatch`). I created a release for the 0.2.7 version.

**Review by swift-kim — APPROVED** [review]
Thanks!

**Pull request merged**